tasks/deploy-static.js

```diff
@@ -18,6 +18,10 @@ function task (opts) {
 	const destination = opts.destination || "";
 	const bucket = opts.bucket;
 
+	if (files.length < 1) {
+		console.warn("Warning: No files found for upload to s3.  (Directories are ignored)");
```

Comment:
`return Promise.reject('No files found for upload to s3.  (Directories are ignored)');`